test/extern/fnPtrs/passBadFnsToCChapel-cPtrToChapel.good

```diff
@@ -0,0 +1,2 @@
+passBadFnsToCChapel.chpl:11: In function 'baz':
+passBadFnsToCChapel.chpl:12: error: Can't call a C function pointer within Chapel
```

Comment:
Is this a case we'd like to support going forward and was just too difficult, or is it something you don't think we should support?